ewah/ewah_utils/yml_loader.py

```diff
@@ -15,12 +18,15 @@
 Accessed 2020-07-28
 """
 
+with create_session() as session:
+    af_val = {var.val for var in session.query(Variable)}
```

Comment:
Request (blocking): I think it would be better to read the airflow variables in the `__init__` method below, please move those lines accordingly :)

Reply:
Also: Why are you using a `set` here? Wouldn't it make more sense to create a `dict`?